gluster-exporter/metric_volume.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	volumeHealLabels = []MetricLabel{
+		{
+			Name: "volume",
+			Help: "Volume Name",
+		},
+		{
+			Name: "brick_path",
+			Help: "Brick Path",
+		},
+		{
+			Name: "host",
+			Help: "Hostname",
+		},
+	}
+
+	glusterVolumeHealCount = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "volume_heal_count",
+		Help:      "self heal count for volume",
+		LongHelp:  "",
+		Labels:    volumeHealLabels,
+	})
+)
+
+func getVolumeHealLabels(volname string, host string, brick string) prometheus.Labels {
+	return prometheus.Labels{
+		"volume":     volname,
+		"brick_path": brick,
+		"host":       host,
+	}
+
+}
+
+func healCounts() error {
+	isLeader, err := gluster.IsLeader()
+	if !isLeader || err != nil {
```

Comment:
This message will be logged if current exporter is not leader.

```
if err != nil {
    // Log
    return err
}
if !isLeader {
    return nil
}
```

Reply:
ack. will fix this